capstone/backend/src/main/java/com/google/sps/servlets/CreateOrUpdateUserlistServlet.java

```diff
@@ -0,0 +1,78 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that creates or updates user lists.
+ */
+@WebServlet("/api/v1/create-or-update-user-list-servlet")
+public class CreateOrUpdateUserlistServlet extends HttpServlet {
+  private static final int BAD_REQUEST = 400;
+  private static final int INTERNAL_SERVER_ERROR = 500;
+  private static final int OK = 200;
+
+  private class RequestBody {
+    private int userId;
+    private UserList userList;
+  }
+
+  private class UserList {
+    private int listId;
+    private String displayName;
+    private List<String> itemTypes;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson g = new Gson();
+    SpannerUtilFunctions suf = new SpannerUtilFunctions();
```

Comment:
This is not implemented yet right?

Reply:
It is sort of implemented. I pushed the file to the DatabaseWrapperFunctions branch when I moved the database specific functions out of the servlets into a separate class since I already wrote the functions. I wasn't sure if Brett wanted to use what I had or change it so I didn't create a pull request.

Reply:
Ok, probably good to sync up with Brett on this.

Reply:
Yeah, I messaged him about it.